[turborepo] Can't Run benchmark

### What version of Turborepo are you using?

main

### What package manager are you using / does the bug impact?

Yarn v1

### What operating system are you using?

Mac

### Describe the Bug

Following the instructions on the benchmark readme, I go to the client file and run `make turbo-prod`, with the following results:

```
make turbo-prod
/bin/sh: go: command not found
/bin/sh: go: command not found
cargo build --release --manifest-path ../crates/turborepo/Cargo.toml
warning: profiles for the non root package will be ignored, specify profiles at the workspace root:
package:   /Users/scottpierce/workspace/repositories/vercel-turbo/crates/turborepo-glob-match/Cargo.toml
workspace: /Users/scottpierce/workspace/repositories/vercel-turbo/Cargo.toml
   Compiling directories v4.0.1
   Compiling sha2 v0.10.6
   Compiling dirs-next v2.0.0
   Compiling globwatch v0.1.0 (/Users/scottpierce/workspace/repositories/vercel-turbo/crates/turborepo-globwatch)
   Compiling turbopath v0.1.0 (/Users/scottpierce/workspace/repositories/vercel-turbo/crates/turborepo-paths)
   Compiling node-semver v2.1.0
   Compiling turbo-updater v0.1.0 (/Users/scottpierce/workspace/repositories/vercel-turbo/crates/turborepo-updater)
   Compiling tonic v0.8.3
   Compiling webbrowser v0.8.8
   Compiling turborepo-lib v0.1.0 (/Users/scottpierce/workspace/repositories/vercel-turbo/crates/turborepo-lib)
error[E0554]: `#![feature]` may not be used on the stable release channel
 --> crates/turborepo-paths/src/lib.rs:1:12
  |
1 | #![feature(assert_matches)]
  |            ^^^^^^^^^^^^^^

error[E0554]: `#![feature]` may not be used on the stable release channel
  --> crates/turborepo-globwatch/src/lib.rs:20:12
   |
20 | #![feature(drain_filter)]
   |            ^^^^^^^^^^^^

For more information about this error, try `rustc --explain E0554`.
error: could not compile `turbopath` due to previous error
warning: build failed, waiting for other jobs to finish...
error: could not compile `globwatch` due to previous error
make: *** [turbo-prod] Error 101
```

### Expected Behavior

It works.

### To Reproduce

I have an Apple M1 Max.

### Reproduction Repo

_No response_

--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -9,7 +9,7 @@ Thanks for your interest in contributing to Turbo!
     - [Building Turborepo](#building-turborepo)
     - [TLS Implementation](#tls-implementation)
     - [Running Turborepo Tests](#running-turborepo-tests)
-      - [Go Tests](#go-tests)
+      - [Turborepo Tests](#turborepo-tests)
   - [Debugging Turborepo](#debugging-turborepo)
   - [Benchmarking Turborepo](#benchmarking-turborepo)
   - [Updating `turbo`](#updating-turbo)
@@ -58,7 +58,7 @@ On macOS:
 brew install moreutils jq zstd # (moreutils is for sponge)
 ```
 
-#### Go Tests
+#### Turborepo Tests
 
 First: `npm install -g turbo`.
 
@@ -106,8 +106,7 @@ Then from the root directory, you can run:
 
 ## Benchmarking Turborepo
 
-1. `make turbo-prod`
-2. From the `benchmark/` directory, run `pnpm run benchmark`.
+Follow the instructions in the [`benchmark/README.md`](./benchmark/README.md).
 
 ## Updating `turbo`
 

--- a/benchmark/README.md
+++ b/benchmark/README.md
@@ -2,5 +2,6 @@
 
 To run benchmarks for turborepo
 
-1. From `../cli/` run `make turbo-prod` to build turbo
-2. From this directory `pnpm run benchmark`
+1. Follow the [Building Turborepo](../CONTRIBUTING.md#building-turborepo) instructions to install dependencies
+2. From `../cli/` run `make turbo-prod` to build turbo
+3. From this directory `pnpm run benchmark`

